test/e2e/gateway_api_test.go

```diff
@@ -273,6 +279,18 @@ func ensureCRDs(t *testing.T) {
 	}
 }
 
+// ensureExperimentalCRDs tests that experimental Gateway API custom resource definitions exist.
+func ensureExperimentalCRDs(t *testing.T) {
```

Comment:
This is identical to `ensureCRDs`, so why not just parameterize the crd names?

Reply:
I returned to [the previous idea of using `ensureExperimentalCRDs`](https://github.com/openshift/cluster-ingress-operator/pull/1200#discussion_r2005192321), so this function doesn't exist anymore.